pkg/app/init.go

```diff
@@ -0,0 +1,203 @@
+package app
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+	"go.uber.org/zap"
+	"helm.sh/helm/v3/pkg/cli"
+
+	"github.com/helmfile/helmfile/pkg/helmexec"
+)
+
+const (
+	HelmRequiredVersion     = "v2.10.0"
+	HelmRecommendedVersion  = "v3.10.1"
+	HelmDiffRequiredVersion = "v3.4.0"
+	HelmInstallCommand      = "https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3"
+)
+
+var manuallyInstallCode = 1
```

Comment:
use const.

Reply:
Address references to constants are not allowed in golang